ui/src/reusable_ui/types/index.ts

```diff
@@ -43,3 +50,71 @@ export enum Greys {
   Ghost = '#fafafc',
   White = '#ffffff',
 }
+
+export enum IconFont {
+  Alerts = 'alerts',
+  AuthZero = 'authzero',
+  BarChart = 'bar-chart',
+  Capacitor = 'capacitor2',
+  CaretDown = 'caret-down',
+  CaretLeft = 'caret-left',
+  CaretRight = 'caret-right',
+  CaretUp = 'caret-up',
+  Checkmark = 'checkmark',
+  Circle = 'circle',
+  Clock = 'clock',
+  CogOutline = 'cog-outline',
+  CogThick = 'cog-thick',
+  Collapse = 'collapse',
+  CrownOutline = 'crown-outline',
+  CrownSolid = 'crown2',
+  Cube = 'cube',
+  Cubouniform = 'cubo-uniform',
+  DashF = 'dash-f',
+  DashH = 'dash-h',
+  DashJ = 'dash-j',
+  Disks = 'disks',
+  Download = 'download',
+  Duplicate = 'duplicate',
+  ExpandA = 'expand-a',
+  ExpandB = 'expand-b',
+  Export = 'export',
+  Eye = 'eye',
+  EyeClosed = 'eye-closed',
+  EyeOpen = 'eye-open',
+  GraphLine = 'graphline2',
+  Group = 'group',
+  Heroku = 'heroku',
+  herokusimple = '',
+  Import = 'import',
+  OAuth = 'oauth',
+  Octagon = 'octagon',
+  Okta = 'okta',
+  Pause = 'pause',
+  Play = 'play',
+  Plus = 'plus',
+  Pulse = 'pulse-c',
+  Refresh = 'refresh',
+  Remove = 'remove',
+  Search = 'search',
+  Server = 'server2',
+  Shuffle = 'shuffle',
+  Square = 'square',
+  TextBlock = 'text-block',
+  Trash = 'trash',
+  Triangle = 'triangle',
+  User = 'user',
+  UserAdd = 'user-add',
+  UserOutline = 'user-outline',
+  UserRemove = 'user-remove',
+  Wood = 'wood',
+  Wrench = 'wrench',
+  AlertTriangle = 'alert-triangle',
```

Comment:
what do you think about making this fully alphabetically sorted? could help with later additions and general readability/dev-friendliness

Reply:
Sure! Is there an automated way to do that?